### PR TITLE
Update raspberry pi model to module

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tutorials/get-started/servo-mousemover.md [hunk 1]
```diff
@@ -175,7 +175,9 @@ Create a [board component](/components/board/):
 
 1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-1. Select the `board` type, then select the `pi` model.
+1. Select the `board` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+   If you are using a Raspberry Pi 5, user the `pi5` model.
```
Comment (sguequierre) [inline docs/tutorials/get-started/servo-mousemover.md:179]:
```suggestion
   If you are using a Raspberry Pi 5, use the `pi5` model.
```
is this accurate? there is no servo pi5 model

File: docs/tutorials/get-started/blink-an-led.md [hunk 2]
```diff
@@ -137,7 +137,8 @@ Go to the [Viam app](https://app.viam.com/) and navigate to your new machine's *
 
 Add a [_board component_](/components/board/) to represent your single-board computer, which in this case is the Raspberry Pi.
 To create the new component, click the **+** icon next to your machine {{< glossary_tooltip term_id="part" text="part" >}} in the left-hand menu and select **Component**.
-Select the `board` type, then select the `pi` model.
+Select the `board` type, then select the `viam:raspberry-pi:rpi` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+If you are using a Raspberry Pi 5, user the `pi5` model.
```
Comment (sguequierre) [inline docs/tutorials/get-started/blink-an-led.md:141]:
```suggestion
If you are using a Raspberry Pi 5, use the `pi5` model.
```

File: docs/tutorials/projects/claw-game.md [hunk 3]
```diff
@@ -168,7 +168,8 @@ Navigate to the **Config** tab of your machine's page and select your main part
 Click the **Components** subtab.
 Click the **Create component** button in the lower-left corner.
 
-Add your [board](/components/board/) with type `board` and model `pi`.
+Add your [board](/components/board/) with type `board` and model `viam:raspberry-pi:rpi` if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+If you are using a Raspberry Pi 5, user the `pi5` model.
```
Comment (sguequierre) [inline docs/tutorials/projects/claw-game.md:172]:
```suggestion
If you are using a Raspberry Pi 5, use the `pi5` model.
```

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 4]
```diff
@@ -214,7 +214,8 @@ Now, configure your rover to:
 To configure your [servo](/components/servo/), go to your rover's **CONFIGURE** tab.
 
 - Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-- Select the `servo` type, then select the `pi` model (since you've attached your servo to a Raspberry Pi).
+- Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+  If you are using a Raspberry Pi 5, user the `pi5` model.
```
Comment (sguequierre) [inline docs/tutorials/projects/integrating-viam-with-openai.md:218]:
```suggestion
  If you are using a Raspberry Pi 5, use the `pi5` model.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md [hunk 5]
```diff
@@ -184,7 +184,8 @@ First, add your Pi as a [board component](/components/board/):
 {{% tab name="Config Builder" %}}
 
 Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-Select the `board` type, then select the `pi` model.
+Select the `board` type, then select the `viam:raspberry-pi:pi` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+If you are using a Raspberry Pi 5, user the `pi5` model.
```
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:188]:
```suggestion
If you are using a Raspberry Pi 5, use the `pi5` model.
```

**Review by sguequierre — CHANGES_REQUESTED** [review]

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 6]
```diff
@@ -214,7 +214,8 @@ Now, configure your rover to:
 To configure your [servo](/components/servo/), go to your rover's **CONFIGURE** tab.
 
 - Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-- Select the `servo` type, then select the `pi` model (since you've attached your servo to a Raspberry Pi).
+- Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+  If you are using a Raspberry Pi 5, use the `pi5` model.
```
Comment (npentrel) [inline docs/tutorials/projects/integrating-viam-with-openai.md:218]: @susmitaSanyal for servos, do we have a model we recommend for pi5?

File: docs/tutorials/get-started/servo-mousemover.md (hunk 1 above)
Comment (susmitaSanyal) [reply docs/tutorials/get-started/servo-mousemover.md]: yes, that's correct.

File: docs/tutorials/projects/integrating-viam-with-openai.md (hunk 6 above)
Comment (npentrel) [inline docs/tutorials/projects/integrating-viam-with-openai.md:218]:
```suggestion
  If you are using a Raspberry Pi 5, use the `gpio` model.
```

File: docs/tutorials/get-started/servo-mousemover.md [hunk 7]
```diff
@@ -188,7 +190,7 @@ Create a [servo component](/components/servo/):
 
 1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-1. Select the `servo` type, then select the `pi` model.
+1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model.
```
Comment (npentrel) [inline docs/tutorials/get-started/servo-mousemover.md:193]:
```suggestion
1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 3, 4 or Zero 2 W.
   If you are using a Raspberry Pi 5, use the `gpio` model.
```

**Comment by susmitaSanyal** [comment]
Screenshot 2024-09-25 at 10 21 42 AM
@npentrel this  can be deleted and/or replaced with the module

File: docs/tutorials/get-started/servo-mousemover.md [hunk 8]
```diff
@@ -188,7 +190,8 @@ Create a [servo component](/components/servo/):
 
 1. Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 1. Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-1. Select the `servo` type, then select the `pi` model.
+1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 3, 4 or Zero 2 W.
+   If you are using a Raspberry Pi 5, use the `gpio` model.
```
Comment (susmitaSanyal) [inline docs/tutorials/get-started/servo-mousemover.md:194]:
```suggestion
1. Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model.
```

Comment (susmitaSanyal) [reply docs/tutorials/get-started/servo-mousemover.md:194]: apologies, pi-servo should work with any board.

Comment (npentrel) [reply docs/tutorials/get-started/servo-mousemover.md:194]: awesome thanks!

File: docs/tutorials/projects/integrating-viam-with-openai.md [hunk 9]
```diff
@@ -214,7 +214,8 @@ Now, configure your rover to:
 To configure your [servo](/components/servo/), go to your rover's **CONFIGURE** tab.
 
 - Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
-- Select the `servo` type, then select the `pi` model (since you've attached your servo to a Raspberry Pi).
+- Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model if you are using a Raspberry Pi 4, Raspberry Pi 3 or Raspberry Pi Zero 2 W.
+  If you are using a Raspberry Pi 5, use the `gpio` model.
```
Comment (npentrel) [inline docs/tutorials/projects/integrating-viam-with-openai.md:218]:
```suggestion
- Select the `servo` type, then select the `viam:raspberry-pi:pi-servo` model.
```

**Pull request merged**